hw09-orm/src/main/java/ru/otus/homework/jdbc/JdbcTemplateImpl.java

```diff
@@ -0,0 +1,224 @@
+package ru.otus.homework.jdbc;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+import ru.otus.homework.annotation.Id;
+import ru.otus.homework.api.service.DbServiceException;
+import ru.otus.homework.jdbc.sessionmanager.SessionManagerJdbc;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Savepoint;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class JdbcTemplateImpl implements JdbcTemplate {
```

Comment:
Интерфейс JdbcTemplate с дженериками:
```java
public interface JdbcTemplate<T>
```


```suggestion
public class JdbcTemplateImpl<T> implements JdbcTemplate<T> {
```

И методы сделайте через дженерики:
```java
public interface JdbcTemplate<T> {
    void create(T objectData);

    void update(T objectData);

    void createOrUpdate(T objectData);

    <T> T load(long id, Class<T> clazz);
}
```